server.js

```diff
@@ -142,24 +142,18 @@ app.get('/intranet', function(req, res) {
   }
 
   request({
-    url: `${SERVICES_URL}/credits/users/${req.session.student.netid}`,
+    url: `${SERVICES_URL}/merch/users/${req.session.student.netid}`,
```

Comment:
Yeah, I thought it was a good way to avoid the UI making requests each time.

The user model in merch "encodes" the balance since its needed for transactions, so I thought it made sense to follow through with that everywhere and make use of that.